classes/class-log.php

```diff
@@ -140,7 +133,7 @@ function ( $var ) {
 			'connector' => (string) $connector,
 			'context'   => (string) $context,
 			'action'    => (string) $action,
-			'ip'        => (string) $this->ip_address,
+			'ip'        => (string) $ip_address,
```

Comment:
Both the boolean `false` as well as the `null` will be converted into an empty string here. Is that the intended behavior?